zap/src/main/java/org/zaproxy/zap/utils/ZapNumberSpinner.java

```diff
@@ -51,10 +56,17 @@ public ZapNumberSpinner(int minValue, int defaultValue, int maxValue) {
             this.defaultValue = defaultValue;
         }
         setModel(new SpinnerNumberModel(this.defaultValue, minValue, maxValue, 1));
-        ((NumberEditor) getEditor())
-                .getTextField()
+        JTextField textField = ((NumberEditor) getEditor()).getTextField();
```

Comment:
Could be `JFormattedTextField` and avoid the cast.

Reply:
DOH